### PR TITLE
Sign Docker images using cosign

### PR DESCRIPTION
**Pull request by gaelgatelement**
---
Here's what your changelog entry will look like:

## ✨ Features
 * Sign Docker images using cosign ([\#26817](https://github.com/element-hq/element-web/pull/26817)).

### CODE REVIEW

**Review by t3chguy — CHANGES_REQUESTED** [review]
Please provide context, or an issue which provides context.

**Comment by gaelgatelement** [comment]
> Please provide context, or an issue which provides context.

Context : https://github.com/element-hq/sre-internal/issues/2920#issue-2009797939

File: .github/workflows/dockerhub.yaml [hunk 1]
```diff
@@ -66,6 +73,17 @@ jobs:
...
                   labels: ${{ steps.meta.outputs.labels }}
 
+            - name: Sign the images with GitHub OIDC Token
+              env:
+                  DIGEST: ${{ steps.build-and-push.outputs.digest }}
+                  TAGS: ${{ steps.meta.outputs.tags }}
+              run: |
+                  images=""
+                  for tag in ${TAGS}; do
+                    images+="${tag}@${DIGEST} "
```
Comment (t3chguy) [inline .github/workflows/dockerhub.yaml:83]:
```suggestion
                      images+="${tag}@${DIGEST} "
```

**Review by t3chguy — APPROVED** [review]
Looks sane otherwise

**Pull request merged**